src/main/java/com/gmail/shablinski93/repository/impl/ReadFileRepositoryImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.gmail.shablinski93.repository.impl;
+
+import com.gmail.shablinski93.repository.ConnectionRepository;
+import com.gmail.shablinski93.repository.ReadFileRepository;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFileRepositoryImpl implements ReadFileRepository {
+    private final ConnectionRepository connectionRepository = ConnectionRepositoryImpl.getInstance();
+
+    @Override
+    public void getListStringsOfOriginalFile(String nameFile) {
```

Comment:
method name is confusing a little bit, it closer to some data initiation, than getting list